docs/zh/latest/plugin-develop.md

```diff
@@ -41,6 +41,7 @@ Apache APISIX 提供了两种方式来添加新的功能。
 │           └── plugins
 │               └── 3rd-party.lua
 ```
+**注意：自定义插件根目录下必须存在 `/apisix/plugins` 的子目录**
```

Comment:
```suggestion
:::note

自定义插件根目录下必须存在 `/apisix/plugins` 的子目录。

:::
```

Reply:
I don't understand the meaning of this sentence, could you describe it in detail?

Reply:
If the plug-in directory is user-defined, there must be a directory of '/ apifix / plugins' under the user-defined root directory, so that apifix can recognize the plug-in

Reply:
感觉应该写:   插件自定义根目录下必须存在 `/apisix/plugins` 的子目录    @hf400159

Reply:
如果你需要自定义插件的目录，请在该目录下创建 `/apisix/plugins` 的子目录。